src/go/rpk/pkg/cli/cmd/cloud/logout.go

```diff
@@ -20,7 +20,7 @@ func NewLogoutCommand(fs afero.Fs) *cobra.Command {
 	return &cobra.Command{
 		Use:   "logout",
 		Short: "Logout from vectorized cloud",
-		Long:  `If there's token stored locally after you've logged in, it will be wiped out.`,
+		Long:  `If there's a token stored locally after you've logged in, it will be wiped out.`,
```

Comment:
The change in this line should be squashed onto the previous one

Reply:
fixed